styles.css

```diff
@@ -331,33 +345,293 @@ input,
   border-color: var(--primary-t-green);
 }
 
-#contact-form > button {
+#contact-form button {
   border: none;
 }
 
-.contact-section > ul {
+#contact-section ul {
   display: flex;
   justify-content: center;
   align-items: center;
   gap: 1.25rem;
 }
 
-.contact-section > span {
+#contact-section span {
   display: block;
   margin-top: 0.8rem;
   color: rgb(66, 82, 110);
 }
 
-.contact-bg {
-  position: relative;
-}
-
-.contact-bg > img {
+#contact-bg {
   position: absolute;
-  right: -1.05rem;
-  top: -8rem;
+  right: 0;
+  bottom: 19rem;
 }
 
 #contact-line-break {
   margin: 15rem auto 0;
 }
+
+/* Footer */
+footer {
+  display: none;
+}
+
+/* DESKTOP VERSION */
+
+@media screen and (min-width: 768px) {
+  header {
+    position: absolute;
+    background-color: transparent;
+    justify-content: center;
+  }
+
+  .logo-container {
+    display: none;
+  }
+
+  #hamburger {
+    display: none;
+  }
+
+  nav > ul {
+    height: 6rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 1.25rem;
+  }
+
+  .nav-menu {
+    display: block;
+    padding: 0.5rem 1rem;
+    color: rgb(25, 28, 28);
+    font-weight: 600;
+  }
+
+  .headline {
+    margin-top: 0;
+    width: 100%;
+    height: 100vh;
+    background: url(icon-img/desktop/Header-illustration-desktop.svg) no-repeat;
+    background-size: 100%;
+    justify-content: flex-start;
+    gap: 2.5rem;
+  }
```

Comment:
- [ ] Greet job @mwismann , I really never see a design beautifully made like this , however according to the requirement you should stick as much as possible to the Figma design, though make sure that the missing details on this picture are completed 

### Figma design 

![Screenshot from 2023-01-20 00-22-53](https://user-images.githubusercontent.com/80458199/213576612-6d17d8fd-8eb9-42ef-a7ed-0d0056a36c20.png)


### Your design

![Screenshot from 2023-01-20 00-23-41](https://user-images.githubusercontent.com/80458199/213576657-1b49695a-0199-441b-98ef-7505f9ed8b5d.png)

Reply:
Hi @iambenkis ,

I really appreciate your words and advice! 🙏

I just have a question regarding the design. As you can see in the image below, the background image works for me, even in different screen sizes 😖 

### **Design on my browser**
![image](https://user-images.githubusercontent.com/107666488/213586009-7b18e515-aecd-4200-8e28-75a9db1f2d54.png)

### **With Inspect**
![image](https://user-images.githubusercontent.com/107666488/213586381-fd28c836-1e89-45c8-b1b5-a37e1fbe6e06.png)